app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Graphical tool to show free disk space like df"
+HOMEPAGE="https://gitlab.com/mazes_80/gtkdiskfree"
+SRC_URI="https://gitlab.com/mazes_80/${PN}/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="gtk2 old-color-selector"
+
+DEPEND="
+	dev-libs/glib
+	gtk2? ( x11-libs/gtk+:2 )
+	!gtk2? ( x11-libs/gtk+:3 )
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf"
+RDEPEND="${DEPEND}"
+BDEPEND="sys-devel/gettext
+	virtual/pkgconfig"
+
+src_configure() {
+	econf $(use_with gtk2) \
+		$(use gtk2 || use_enable old-color-selector)
```

Comment:
I wouldn't introduce a new USE flag for this. You can simply control it with `gtk2` IUSE.

It's also a bit weird that `-gtk2` pulls in gtk3, but uses something called **old**-color-selector?

Reply:
The old color selector is superior than the one used with the GTK3 version, don't know why it is either.